build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml

```diff
@@ -8,7 +8,7 @@ steps:
   - label: "build"
     commands:
       - "git submodule sync && git submodule update --init --jobs 8 --depth 1"
-      - "docker run --user=$(id -u):$(id -g) --volume=\\$PWD:\\$IREE_DOCKER_WORKDIR --workdir=\\$IREE_DOCKER_WORKDIR --rm gcr.io/iree-oss/android@sha256:9bc723fc707a18bd0c1be9c12e01ea5bb7c7d77f607427879e10ffcffd7d2bb5 build_tools/cmake/build_android.sh arm64-v8a"
+      - "docker run --user=$(id -u):$(id -g) --volume=\\$PWD:\\$IREE_DOCKER_WORKDIR --workdir=\\$IREE_DOCKER_WORKDIR --rm gcr.io/iree-oss/android@sha256:9bc723fc707a18bd0c1be9c12e01ea5bb7c7d77f607427879e10ffcffd7d2bb5 build_host_and_android.sh arm64-v8a"
```

Comment:
whoops, this should have kept the `build_tools/` part of the path
https://buildkite.com/iree/iree-test-android/builds/6875

Reply:
Sent a fix: https://github.com/iree-org/iree/pull/10037

Reply:
Huh, whoops. Bad find-replace I guess. Thanks for catching